cfgov/v1/templates/v1/images_report.html

```diff
@@ -0,0 +1,73 @@
+{% extends 'wagtailadmin/reports/base_report.html' %}
+{% load i18n wagtailadmin_tags %}
+
+{% block actions %}
+    {% if view.list_export %}
+        <div class="actionbutton">
+            <a href="{{ view.csv_export_url }}" class="button bicolor button--icon">{% icon name="download" wrapped=1 %}{% trans 'Download CSV' %}</a>
+        </div>
+    {% endif %}
+{% endblock %}
+
+{% block results %}
+    {% if object_list %}
+        <table class="listing">
+            <thead>
+                <tr>
+                    <th>
+                        Title
+                    </th>
+                    <th>
+                        File
+                    </th>
+                    <th>
+                        Size (bytes)
```

Comment:
It might be worth it to do a little processing and spit out file sizes in KBs for easier readability instead.

Reply:
I'd actually rather keep them in bytes at least for now, just to understand the usefulness of the field in the export and data review before doing too much extra work.